WcaOnRails/app/views/polls/_polls_list.html.erb

```diff
@@ -21,6 +21,9 @@
       <% if current_user.can_create_poll? %>
         <td><%= link_to "Edit", edit_poll_path(poll), class: "btn btn-xs btn-success" %></td>
       <% end %>
+      <td>
+        <%= poll.over? ? "Voting closed on" : "Open until" %> <%= poll.deadline %>
```

Comment:
How about using `wca_local_time` here? For example: https://github.com/jfly/worldcubeassociation.org/blob/756402ded19b398f3c6cee2dafa915db1e8a1483/WcaOnRails/app/views/posts/_post.html.erb#L26